src/Perpetuum/Players/Player.cs

```diff
@@ -721,7 +721,12 @@ public override void OnAggression(Unit victim)
             if (victim is ITaggable taggable)
                 taggable.Tag(this,TimeSpan.Zero);
 
-            if ( victim is MobileTeleport || victim is BlobEmitterUnit || victim is IPBSObject || victim is WallHealer || victim is ProximityProbeBase)
+            if (victim is BlobEmitterUnit b && b.IsPlayerSpawned)
+            {
+                ApplyPvPEffect();
+                return;
+            }
+            else if (victim is MobileTeleport || victim is IPBSObject || victim is WallHealer || victim is ProximityProbeBase)
```

Comment:
I would merge these 2 seeing as they have the same conclusion

Reply:
Yeah, guess I'll do some line wrapping, it was just getting too long for my tastes..

Probably should put in a linter one day...